CorsixTH/Lua/languages/english.lua

```diff
@@ -684,6 +684,7 @@ hotkey_window = {
   ingame_recallPosition_8 = "8",
   ingame_recallPosition_9 = "9",
   ingame_recallPosition_0 = "10",
+  back = "Back",
```

Comment:
Line not needed here. I think you meant `options_window.back` but that already exists (line 473)